app/src/main/java/com/example/ihonboarding/feature/home/data_source/local/NewsLocalDataSource.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.ihonboarding.feature.home.data_source.local
+
+import com.example.ihonboarding.domain.home.model.News
+import kotlinx.coroutines.flow.Flow
+
+interface NewsLocalDataSource {
+    fun getLocalNewsList(): Flow<List<News>>
```

Comment:
Why you mix between flow and suspend function here? Just want to know the consideration

Reply:
I just always use the "suspend" to do the API call. And it returns the flow containing the whole process of fetching data from local database and/or the remote data source.